Events/index.md

```diff
@@ -5,19 +5,19 @@ author: Doug, Jackson, & Tony
 ---
 
 # Upcoming Conferences
-* June 17-21, 2017 - [2018 American Nuclear Society Annual Meeting](http://www.ans.org/meetings/file/766)
-  - Summaries due January 12, 2018
-* January 27, 2018 - [2018 Oregon State University Student Winter Conference](http://groups.engr.oregonstate.edu/ANS/content/annual-winter-conference-0)
-  - Abstracts due December 8, 2017
-* February 8, 2018 - [2018 Graduate Research Showcase](http://engineering.oregonstate.edu/2018-graduate-research-showcase)
-  - Register by November 20, 2017
 * April 5-7, 2018 - [2018 American Nuclear Society Student Conference](http://www.ansstudentconference2018.com/)
-  - Abstracts due (typically January)
+* June 17-21, 2018 - [2018 ANS Annual Meeting](http://ansannual.org/)
+  - Philadelphia, PA
+* June 9-13, 2019 - [2019 ANS Annual Meeting](http://ansannual.org/)
```

Comment:
same link for two different meetings?

Reply:
they only use one url for their annual meetings, they just update the page to reflect the soonest upcoming meeting. So, that *will* be the url. I'll delete it.

Reply:
yeah, I hear you.  just might be confusing. I think leaving it linkless until that link represents that meeting is the best move.